### ISSUE TITLE
Could not pass event AsyncPlayerChatEvent to ChatEx v2.5.2

### ISSUE THREAD

**Issue by lerokko** [TheJeterLP/ChatEx#19]
Sorry if this is not an issue with your plugin.
Today a player joined and sometimes when he send messages they where malformed. The current format is "[N] playername: message" without the <> from vanilla. For some reason some of his messages (2 to be exact both of them only saying ":p") appeared with the two brackets.

I'm running paperspigot build 332 with luckperms, vault, vault chat formatter.

```
[Tue, 2. Jun 2020 18:41:29 CEST INFO] [N] redacted3: bye cruel world
[Tue, 2. Jun 2020 18:41:23 CEST ERROR] Could not pass event AsyncPlayerChatEvent to ChatEx v2.5.2
[Tue, 2. Jun 2020 18:41:23 CEST INFO] <[N] redacted3> :p
```

**Comment by TheJeterLP** [comment]
Do you have any more error information? Or was there no error code?



lerokko <notifications@github.com> schrieb am Di., 2. Juni 2020, 19:43:

**Comment by lerokko** [comment]
This is how it appeared in the console. I included the first line to show you how it usually appears. 2nd line is the Error message which I assume causes the malformed message (line 3).

If there are any other logs I could provide, please let me know. Right know this is sadly all I got.
On top of that I was not able to reproduce it.

**Comment by TheJeterLP** [comment]
Weird, I will check it later

lerokko <notifications@github.com> schrieb am Mi., 3. Juni 2020, 17:19:

**Comment by TheJeterLP** [comment]
Should be fixed with the new 2.5.3 version, thanks!

**Comment by lerokko** [comment]
**Thank you for the swift fix!** I just updated to 2.5.5. (I hope my reply does not reopen this, I don't know how github works, if so sorry)

**Comment by TheJeterLP** [comment]
No it shouldn't reopen, no problem :)

lerokko <notifications@github.com> schrieb am Fr., 5. Juni 2020, 12:33:
